resource/ui/ts/models/status.ts

```diff
@@ -0,0 +1,218 @@
+                }
+                return m("div",[
+                    m("table", [
+                      m("tr", [m("td", "Node Id:"), m("td", m("a[href=/nodes/" + store.desc.node.node_id + "]", { config: m.route }, store.desc.node.node_id))]),
+                      m("tr", [m("td", "Node Network:"), m("td", store.desc.node.address.network)]),
+                      m("tr", [m("td", "Node Address:"), m("td", store.desc.node.address.address)]),
+                      m("tr", [m("td", "Started at:"), m("td", _formatDate(store.started_at))]),
+                      m("tr", [m("td", "Updated at:"), m("td", _formatDate(store.updated_at))]),
+                      m("tr", [m("td", "Ranges:"), m("td", store.range_count)]),
+                      m("tr", [m("td", "Leader Ranges:"), m("td", store.leader_range_count)]),
+                      m("tr", [m("td", "Available Ranges:"), m("td", store.available_range_count)]),
+                      m("tr", [m("td", "Availablility:"), m("td", _availability(store))]),
+                      m("tr", [m("td", "Under-Replicated Ranges:"), m("td", store.leader_range_count - store.replicated_range_count)]),
+                      m("tr", [m("td", "Fully Replicated:"), m("td", _replicated(store))])
+                    ]),
+                    Stats.CreateStatsTable(store.stats)
+                ]);
+            }
+
+            public AllDetails(): _mithril.MithrilVirtualElement {
+                var status:Proto.Status = {
+                    range_count: 0,
+                    updated_at: 0,
+                    started_at: 0,
+                    leader_range_count: 0,
+                    replicated_range_count: 0,
+                    available_range_count: 0,
+                    stats: Proto.NewMVCCStats()
+                };
+
+                for (var storeId in this._data) {
+                    var storeStatus = this._data[storeId];
+					Proto.AccumulateStatus(status, storeStatus);
+                };
+
+                return m("div", [
+                    m("h2", "Details"),
+                    m("table", [
+                        m("tr", [m("td", "Updated at:"), m("td", _formatDate(status.updated_at))]),
+                        m("tr", [m("td", "Ranges:"), m("td", status.range_count)]),
+                        m("tr", [m("td", "Leader Ranges:"), m("td", status.leader_range_count)]),
+                        m("tr", [m("td", "Available Ranges:"), m("td", status.available_range_count)]),
+                        m("tr", [m("td", "Availablility:"), m("td", _availability(status))]),
+                        m("tr", [m("td", "Under-Replicated Ranges:"), m("td", status.leader_range_count - status.replicated_range_count)]),
+                        m("tr", [m("td", "Fully Replicated:"), m("td", _replicated(status))])
+                    ]),
+                    Stats.CreateStatsTable(status.stats)
+                ]);
+            }
+        }
+
+        export interface NodeStatusResponseSet {
```

Comment:
Everything being returned as an array is wrapped in a 'd'.  I think a generic interface for WrappedResponseSet<T> might make more sense than repeating this same thing so many times.